product_docs/docs/pgd/5/reference/functions.mdx

```diff
@@ -365,6 +365,31 @@ bdr.get_node_sub_apply_lsn(node_name name)
 -   `node_name` &mdash; the name of the node that's the source of the
     replication stream whose LSN is being retrieved.
 
+### `bdr.replicate_ddl_command`
```

Comment:
Thanks for adding this to the reference section. Weird it wasn't present so far.